plugins/source/aws/README.md

```diff
@@ -0,0 +1,350 @@
+# AWS Source Plugin
+
+AWS Source plugin for CloudQuery extract configuration from varietly of AWS APIs and loads them to any of the supported
+CloudQuery Destination plugins
+
+## Links
+
+- Homepage: [https://cloudquery.io](https://cloudquery.io)
+- Documentation: [https://docs.cloudquery.io](https://docs.cloudquery.io)
+- Tables: [./docs/tables/README.md](./docs/tables/README.md)
+- Contributing: [./docs/contributing/README.md](./docs/contributing/README.md)
+- Discord: [https://cloudquery.io/discord](https://cloudquery.io/discord)
+
```

Comment:
Same suggestion as before. Also should we duplicate this information?

Reply:
we can remove the duplicates at just keep tables ? wdyt ? or just keep convention and then we dont need the links.

Reply:
Yeah, maybe keep the links in the README.md?